### PR TITLE
More contributions to the unending lifecycle of bugfixing and the process of avoiding life responsibilities by substituting them for semi-internal validation through the fleeting medium of my work

### PR DESCRIPTION
**Pull request by katef**

### CODE REVIEW

File: src/retest/runner.c [hunk 1]
```diff
@@ -75,6 +75,11 @@ print(const struct fsm *fsm, enum implementation impl,
 		return 0;
 	}
 
+	/* the vmc codegen can emit memcmp() calls */
+	if (impl == IMPL_VMC && fsm_getoptions(fsm)->io == FSM_IO_PAIR) {
+		fprintf(f, "#include <string.h>\n\n");
```
Comment (BinaryVixen899) [inline src/retest/runner.c:80]: @katef is `\n\n` on purpose here?

Comment (katef) [reply src/retest/runner.c:80]: Yep! it's nice to separate headers from the body of code a bit, just a matter of style

**Review by BinaryVixen899 — APPROVED** [review]
As mentioned in Slack, please get another review for the test portions as I am not familiar with this codebase yet. However I've reviewed the rest and it looks good.

**Review by silentbicycle — APPROVED** [review]
lgtm

File: examples/words/main.c [hunk 2]
```diff
@@ -28,7 +28,7 @@ int main(int argc, char *argv[]) {
 	fsm_state_t start;
 	char s[BUFSIZ];
 	int (*dmf)(struct fsm *);
-	void (*print)(FILE *, const struct fsm *);
+	int (*print)(FILE *, const struct fsm *);
```
[the remark this answers is not in the document]
Comment (katef) [reply examples/words/main.c]: There absolutely should! Thanks.
1c1ea19b8125db232207b072e07a4f869ea17e16

**Review by silentbicycle — APPROVED** [review]
Reapproved with typedef. I didn't think that should block approving the PR, but it's good to have.

**Pull request merged**